src/components/CategoryNavContainer/CategoryNav/CategoryNav.scss

```diff
@@ -1,12 +1,99 @@
 @import '~style/variables';
 
+.categoryNav {
+  background-color: $color-white;
+  display: flex;
+  height: 48px;
```

Comment:
This works, but when child items have height set too, setting padding (instead of height) on the parent element means changes to child element height just work.